api/services/workspaces/spec.go

```diff
@@ -0,0 +1,46 @@
+package workspaces
+
+import (
+	"time"
+
+	"github.com/open-sauced/pizza-cli/api/services"
+)
+
+type DbWorkspace struct {
+	ID          string              `json:"id"`
+	CreatedAt   time.Time           `json:"created_at"`
+	UpdatedAt   time.Time           `json:"updated_at"`
+	DeletedAt   *time.Time          `json:"deleted_at"`
```

Comment:
Just checking , but DeletedAt is a pointer as that represents `nil` since this is considered a struct for creating a new DbWorkspace and newly created workspaces have DeletedAt set to `nil`?

Reply:
Yes, in theory, it's a way to represent `null` being sent in the API

Reply:
Although, I'm sure there's rough edges with that implementation in this hand rolled client. Thankfully, `DeletedAt` isn't in the application flow anywhere.